### ISSUE TITLE
Rollup Error caused by require() of JSON file

### ISSUE THREAD

**Issue by lcurrens** [ng-packagr/ng-packagr#325]
## Type of Issue

```
[X] Bug Report
[ ] Feature Request
```

## Description
Getting "Unexpected Token" error in Rollup using ng-packagr.  Error appears to be caused by Rollup attempting to create a Module for a JSON file, which it should be excluding.

The JSON file is referenced like this in file node_modules/statuses/index.js:
`
var codes = require('./codes.json')
` 
That require of codes.json causes the following Unexpected Token error as Rollup tries to parse the JSON file, and tries to create a Module for the JSON file (it should exclude the json file instead).

> BUILD ERROR
> Unexpected token
> Error: Unexpected token
>    at error (C:\GitRepos\uxf-core-CLI\node_modules\rollup\dist\rollup.js:190:14)
>    at Module.error (C:\GitRepos\uxf-core-CLI\node_modules\rollup\dist\rollup.js:14300:3)
>    at tryParse (C:\GitRepos\uxf-core-CLI\node_modules\rollup\dist\rollup.js:14018:10)
>    at new Module (C:\GitRepos\uxf-core-CLI\node_modules\rollup\dist\rollup.js:14062:15)
>   at load.catch.then.then.then.source (C:\GitRepos\uxf-core-> CLI\node_modules\rollup\dist\rollup.js:15964:20)

#### How To Reproduce
Install the "statuses" NPM package:

$ npm install statuses

Use it in a project like this:
var status = require('statuses')

#### Expected Behaviour

Rollup should work in ng-packagr

#### Version Information

```
ng-packagr: v1.6.0
node: v6.10.3
@angular: v4.4.6
rxjs: v5.5.2
zone.js: v0.8.14
```

**Comment by henrahmagix** [comment]
I'm hitting this same problem too. Everything is fine in an app build, but a library build has this error. This means I cannot move some common code from many apps into one library.

Turns out it's because of a comment prepended to the JSON file. I went into the rollup code and logged the JSON that's being transformed:
```json
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
{
  "value": "a string",
  "array": ["an", "array"]
}
```

What is putting that comment there? And why can `@angular-devkit/build-angular:browser` deal with JSON files but `@angular-devkit/build-ng-packagr:build` cannot?
